pixi.yaml

```diff
@@ -42,8 +31,44 @@ update:
     identifier: prefix-dev/pixi
     strip-prefix: v
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility subpackage for Pixi upstream image
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}/usr/local/bin"
+          ln -s ../../bin/pixi "${{targets.subpkgdir}}/usr/local/bin/pixi"
```

Comment:
Just checking... Are we sure on this?

```
usr/local/bin/pixi -> ../../bin/pixi
```

Reply:
It should work but thinking... what if i'm in a dir like `/foo/bar/baz/qux/quux` and run the `/usr/local/bin/pixi` here?